src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
```

Comment:
Seems that you forgot to add `Roboto` font family to your code and it wasn't applied 
```suggestion
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
    <link rel="stylesheet" href="./style.css">
```